README.md

```diff
@@ -58,7 +60,32 @@ yarn install
 yarn dev
 ```
 
-### Starting LocalStack
+### Starting LocalStack and services without dev containers
+
+#### Spare me the details and give me the commands
+
+##### Only do once as part of setup
+
+- You will also have to supply the following environment variables and values to platform-forms-client to get everything working with the local lambdas and localstack.
+
+```shell
+# localstack only simulates a us-east-1 region
+AWS_ACCESS_KEY_ID=test
```

Comment:
Both `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` are not part of my `.env` file but are located in `~/.aws/credentials`.

Reply:
I was running into issues often for both the app and terraform when using the `~/.aws/credentials` .  Especially now since we use rotating Access Keys and Secrets.  I personally found it easier to drop the profiles in `~/.aws/credentials` and just use the env vars.